src/components/Social/Facebook/Facebook.test.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * react libraries
+*/
+import React from 'react';
+/**
+ * third-party libraries
+*/
+import { shallow } from 'enzyme';
+/**
+ * components
+ */
+import FacebookButton from './index';
+/**
+ * Define a shared shallow renderer function
+*/
+const fbButton = (props = {}) => {
+  const facebook = shallow(<FacebookButton {...props} />);
+  return facebook;
+};
+
+/**
+ * Define the tests for the Facebook button
+*/
+describe('Facebook Login', () => {
+/**
+ * Pass the value of the shallow renderer before each test
+*/
+  let facebook;
+  beforeEach(() => {
+    facebook = fbButton();
+  });
+
+  /**
+   * Test to see if it matches the snapshot
+  */
```

Comment:
The is no need for such comments, the statement in the It block should be descriptive enough to tell us what is happening with the test. Kindly remove them